content_scripts/vimium_frontend.coffee

```diff
@@ -229,10 +235,13 @@ getActiveState = ->
 #
 # The backend needs to know which frame has focus.
 #
-registerFocus = ->
-  # settings may have changed since the frame last had focus
-  settings.load()
-  chrome.runtime.sendMessage({ handler: "frameFocused", frameId: frameId })
+registerFocus = (event) ->
+  if event.target == window
+    # Settings may have changed since the frame last had focus.
+    settings.load()
+    # Don't register frameset containers; focusing them is no use.
+    unless document.body?.tagName.toLowerCase() == "frameset"
```

Comment:
> This mean that focusing a `<frameset>`

Can that happen?  If it can, then you're right.

(I guess it could happen if the entire document is a frame set, but the frames don't cover the entire viewport.  Clicking outside of the frame set may then focus it.  Do you have a better example?)

(Separately, if we can focus a `<frameset>`, the in `master` we end up registering the frame set on `focus`, and it becomes part of the `gf` cycle.)